pkg/server/upload.go

```diff
@@ -188,7 +188,7 @@ func (s *uploadServlet) upload(w http.ResponseWriter, r *http.Request) {
 }
 
 func validateKey(ctx context.Context, w http.ResponseWriter, key *pb.TemporaryExposureKey) bool {
-	if key.GetRollingPeriod() != 144 {
+	if key.GetRollingPeriod() < 1 || key.GetRollingPeriod() > 144{
```

Comment:
Is it worth handling the 0 case (reset to 144)? Just in case?

Reply:
There is no setter in the ProtoBuf, based on the docs it says it has to be set, I am going to assume it can't be 0, will keep an eye on it.